cpp/src/Slice/Grammar.y

```diff
@@ -1396,7 +1396,7 @@ interface_list
             else
             {
                 cont->checkIntroduced(scoped->v);
-                intfs->v.push_front(def);
+                intfs->v.push_back(def);
```

Comment:
Another nice effect is now we can use the more natural 'push_back' instead of 'push_front'.